README.md

```diff
@@ -11,19 +11,12 @@ Canal represents the best-of-breed policy-based networking solution for cloud na
 
 Note that the Canal currently uses the Calico and flannel projects as is with no code modifications to either. Canal today is simply a deployment pattern for installing and configuring the projects to work together seamlessly as single network solution from the point of view of the user and orchestration system. In the future the Canal project will likely contribute code changes to Calico and flannel projects to further simplify install and configuration.
 
-## Canal installation instructions
+## Installing
 
-### CoreOS based Kubernetes cluster
-Canal is supported today within the [coreos/coreos-kubernetes](https://github.com/coreos/coreos-kubernetes) repository, which provides step-by-step instructions plus several automated install options to bring up a Kubernetes cluster with Canal networking (Calico policy + flannel VXLAN connectivity by default).
-
-### Kubernetes self-hosted install (Recommended)
-[See here](k8s-install/README.md)
-
-### Step-by-step guide for expert rkt users
-[See here](InstallGuide.md)
-
-### Step-by-step guide for expert users creating their own orchestrator integrations
-[See here](OrchestratorIntegration.md)
+- **CoreOS based Kubernetes cluster:** Canal is supported today within the [coreos/coreos-kubernetes](https://github.com/coreos/coreos-kubernetes) repository, which provides step-by-step instructions plus several automated install options to bring up a Kubernetes cluster with Canal networking (Calico policy + flannel VXLAN connectivity by default).
```

Comment:
First bullet has different formatting than the others - it's bold and has a description.  Can we standardize them?